samples/snippets/schema_test.py

```diff
@@ -339,13 +376,18 @@ def test_create_topic_with_schema_revisions(
     assert f"{proto_schema}" in out
     assert "BINARY" in out or "2" in out
 
+    # Cleanup
+    topic_path = publisher_client.topic_path(PROJECT_ID, PROTO_WITH_REVISIONS_TOPIC_ID)
+    publisher_client.delete_topic(request={"topic": topic_path})
+
 
```

Comment:
This step seems to already exists in https://github.com/googleapis/python-pubsub/blob/93db78d2efcc814992efb8d80e100c5c71a84418/samples/snippets/schema_test.py#L159-L169